src/components/frame/v5/pages/TeamsPage/partials/TeamsPageFilter/TeamsPageFilter.tsx

```diff
@@ -139,8 +139,9 @@ const TeamsPageFilter: FC<TeamsPageFilterProps> = ({
           </div>
           <Header
             title={{
-              id: !isMobile ? 'teamsPage.filter.sortBy' : undefined,
+              id: 'teamsPage.filter.sortBy',
```

Comment:
Great catch!

Reply:
Agree! I can't even understand why that logic was in there in the first place. Thanks for catching this Dave.